pkg/e2etemplates/apiextensions_chart_config_e2e_chart_values.go

```diff
@@ -0,0 +1,17 @@
+package e2etemplates
+
+const ApiextensionsChartConfigE2EChartValues = `chart:
+  channel: {{ .Channel }}
+  configMap:
+    name: {{ .ConfigMapName }}
+    namespace: {{ .ConfigMapNamespace }}
+    resourceVersion: {{ .ConfigMapResourceVersion }}
```

Comment:
I think these inputs should always be scoped via more structure in case the names get longer and prefixes repeat. Like there should be a `ConfigMap` block. Same for `Secret`. But no blocker for me.

Reply:
Do you mean a structure like `.ConfigMap.Name` and so on?
I'll work on that.

Reply:
Yes.

Reply:
Done.